docs/content/contribute/release-process.md

```diff
@@ -19,17 +28,103 @@ The HO has an independent release cadence. For consumer products:
+
+We need also to modify some things manually in the Openshift/Release repo.
+
+## Preparing a release in [Openshift/Release](https://github.com/openshift/release) repository
+
+### Step Registry/CLI
+
+From folder: `ci-operator/step-registry/hypershift/` in the OCP release repo. In each file we will have a section called `from_image:` overriding the current release version (E.G) `"4.11"` to the new one `"4.12"`. You can check an example [here](https://github.com/openshift/release/pull/30297/files):
+
+- `aws/create/hypershift-aws-create-chain.yaml`
+- `aws/destroy-nested-management-cluster/hypershift-aws-destroy-nested-management-cluster-chain.yaml`
+- `aws/destroy/hypershift-aws-destroy-chain.yaml`
+- `aws/setup-nested-management-cluster/hypershift-aws-setup-nested-management-cluster-chain.yaml`
+- `azure/create/hypershift-azure-create-chain.yaml`
+- `azure/destroy/hypershift-azure-destroy-chain.yaml`
+- `dump/hypershift-dump-chain.yaml`
+- `install/hypershift-install-chain.yaml`
+- `kubevirt/create/hypershift-kubevirt-create-chain.yaml`
+
+NEED TO VERIFY THESE ONES
+
+- `aws/run-e2e/external/hypershift-aws-run-e2e-external-chain.yaml`
+- `mce/kubevirt/create/hypershift-mce-kubevirt-create-chain.yaml`
+- `mce/kubevirt/destroy/hypershift-mce-kubevirt-destroy-chain.yaml`
+- `mce/aws/destroy/hypershift-mce-aws-destroy-chain.yaml`
+- `mce/aws/create/hypershift-mce-aws-create-chain.yaml`
+- `agent/create/hostedcluster/hypershift-agent-create-hostedcluster-ref.yaml`
+- `agent/destroy/hypershift-agent-destroy-chain.yaml`
+- `ibmcloud/install/hypershift-ibmcloud-install-chain.yaml`
+
+### Mirroring Images for CPO and HO
+
+For folder `core-services/image-mirroring/hypershift/` we need to add a mapping file for each version we want to mirror. This is a sample about how looks like this kind of [file](https://github.com/openshift/release/blob/master/core-services/image-mirroring/hypershift/mapping_hypershift_4_13). This sample points to version `4.13` and the filename is `mapping_hypershift_4_13`:
+
+```
+registry.ci.openshift.org/ocp/4.13:hypershift quay.io/hypershift/hypershift:4.13
+registry.ci.openshift.org/ocp/4.13:hypershift-operator quay.io/hypershift/hypershift-operator:4.13
+registry.ci.openshift.org/ocp-arm64/4.13:hypershift quay.io/hypershift/hypershift:4.13-arm64 quay.io/hypershift-arm64/hypershift:latest-arm64
+registry.ci.openshift.org/ocp-arm64/4.13:hypershift-operator quay.io/hypershift/hypershift-operator:4.13-arm64
+```
+
+As you can see, the version is hardcoded but, what happens to the `latest` version?:
+
+```
+registry.ci.openshift.org/ocp/4.14:hypershift quay.io/hypershift/hypershift:4.14 quay.io/hypershift/hypershift:latest
+registry.ci.openshift.org/ocp/4.14:hypershift-operator quay.io/hypershift/hypershift-operator:4.14  quay.io/hypershift/hypershift-operator:latest
+registry.ci.openshift.org/ocp-arm64/4.14:hypershift quay.io/hypershift/hypershift:4.14-arm64 quay.io/hypershift-arm64/hypershift:latest-arm64
```

Comment:
```suggestion
registry.ci.openshift.org/ocp-arm64/4.14:hypershift quay.io/hypershift/hypershift:4.14-arm64 quay.io/hypershift/hypershift:latest-arm64
```
That repo was a typo and should be resolved once https://github.com/openshift/release/pull/36933 is merged.

Reply:
Cool, nice catch